src/directives/routeClass.js

```diff
@@ -0,0 +1,28 @@
+class routeClassDirective {
```

Comment:
Will change the structure of this directive so it follows the pattern we decided on in the recent catch up - and update the view directive to do the same.